modules/apps/oauth-client/oauth-client-admin-web/src/main/java/com/liferay/oauth/client/admin/web/internal/portlet/OAuthClientAdminPortlet.java

```diff
@@ -17,7 +17,9 @@
  */
 @Component(
 	property = {
+		"com.liferay.portlet.css-class-wrapper=portlet-oauth2-client-admin",
```

Comment:
@jgarcialr Can you please rename it to `portlet-oauth-client-admin` instead? 
Thx